<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPrincipale</class>
 <widget class="QWidget" name="FenPrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dans ton Chat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="ouvrirSite">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>241</horstretch>
         <verstretch>41</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>246</width>
         <height>47</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>241</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boutonQuote">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>81</horstretch>
         <verstretch>41</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Une quote 
au hasard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonFavori">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>81</horstretch>
         <verstretch>41</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Un favori
 au hasard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonMaj">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>81</horstretch>
         <verstretch>41</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Mise à jour</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonAbout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>81</horstretch>
         <verstretch>41</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>À propos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditQuote">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Quote n°</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelIDQuote">
           <property name="minimumSize">
            <size>
             <width>191</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nombre de quotes :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNbQuotes">
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Aucune</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>47</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Voir la quote n°</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditNumeroQuote"/>
         </item>
         <item>
          <widget class="QPushButton" name="goQuote">
           <property name="text">
            <string>Aller</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkRunStart">
         <property name="text">
          <string>Lancer ce programme au démarrage</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>boutonQuote</tabstop>
  <tabstop>boutonFavori</tabstop>
  <tabstop>boutonMaj</tabstop>
  <tabstop>boutonAbout</tabstop>
  <tabstop>lineEditNumeroQuote</tabstop>
  <tabstop>goQuote</tabstop>
  <tabstop>checkRunStart</tabstop>
  <tabstop>ouvrirSite</tabstop>
  <tabstop>textEditQuote</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>boutonQuote</sender>
   <signal>clicked()</signal>
   <receiver>textEditQuote</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
